programs/fuzz/fuzz_client.c

```diff
@@ -140,7 +140,6 @@ int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
     // mbedtls_ssl_conf_cert_profile, mbedtls_ssl_conf_sig_hashes
 
     srand(1);
```

Comment:
I think that all programs in `programs/fuzz/` used to call `srand(1)` because `dummy_random()` uses `rand()` internally. Now that we removed `dummy_random` we can also remove the call to `srand(1)` above. Wdyt?
This applies to all the programs where `dummy_random` was removed.